pkgs/os-specific/linux/hid-fanatecff/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  kernel,
+  kernelModuleMakeFlags,
+  linuxConsoleTools,
+  nix-update-script,
+}:
+
+let
+  moduleDir = "lib/modules/${kernel.modDirVersion}/kernel/drivers/hid";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hid-fanatecff";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "gotzl";
+    repo = "hid-fanatecff";
+    tag = finalAttrs.version;
+    hash = "sha256-M2jm8pyxHRiswV4iJEawo57GkJ2XOclIo3NxEFgK+q0=";
+  };
```

I would like this change to be merged. A new version of the module has been released since the latest change to this PR, so attaching a suggestion to update that.

```suggestion
  version = "0.2.2";

  src = fetchFromGitHub {
    owner = "gotzl";
    repo = "hid-fanatecff";
    tag = finalAttrs.version;
    hash = "sha256-aVuTnrxw7zWMZ1U21DUKDvcYlIp7iHJHaX8ijmUd/TE=";
  };
```